demo-app/src/components/movie-form/MovieForm.css

```diff
@@ -0,0 +1,92 @@
+.movie-form-container {
```

Comment:
```suggestion
.movie-form-container < form {
```
![image](https://github.com/dhara-basida/react-demo-app/assets/78261193/9b91e8a4-6f52-406e-84d5-23900f8c352d)